engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/deletion/content-view-version-deletion.controller.js

```diff
@@ -109,6 +109,10 @@ angular.module('Bastion.content-views').controller('ContentViewVersionDeletionCo
             return _.pluck($scope.deleteOptions.environments, 'id');
         };
 
+        $scope.selectedEnvironmentNames = function () {
+            return _.pluck($scope.deleteOptions.environments, 'name');
```

Comment:
we no longer use undscore (which had pluck).  Just changing 'pluck' to 'map' makes it work.